### DISCUSSION TITLE
Mechanical Designations and Cleaning

### DISCUSSION THREAD

**Discussion by joaochris** [sws2apps/organized-app#760] [Ideas]
Hello @rhahao. He is well?

I was wondering if it would be possible to add mechanical designations to the CPE, which would include indicators, steering wheels, audio and video. And Kingdom Hall cleaning assignments.

I find CPE very practical. We currently use Google Sheets for both of these assignments. But it would be interesting if we could migrate everything to CPE.

Regarding audio and video, the title and number of columns could be edited if feasible, as other congregations may use different layouts.

As for cleaning, here we use it in groups.

I will attach the two models of these designations that we use.

Audio/Video:
![image](https://github.com/sws2apps/cpe-sws/assets/76891864/0bd637b0-3c56-4538-b3e0-46f613a22904)

Cleaning:
![image](https://github.com/sws2apps/cpe-sws/assets/76891864/121c524d-65f6-4fa6-8744-454b50ae2648)

**Comment by rhahao** [comment]
Hi @joaochris, thanks for sharing. We will add this to our notes.

**Comment by ratara5** [comment]
Thanks a million for this great app!

I just want to share my experience: Our kingdom hall cleaning program is carried out among all congregations that use it. Every week we do: 1) the frequent activity and 2) an additional activity.

[CleaningProgram-ES.pdf](https://github.com/sws2apps/organized-app/files/14793025/PROGRAMA.DE.LIMPIEZA.DEL.SALON.DEL.REINO-1.pdf)

**Reply by rhahao** [reply]
Hi @ratara5, thanks for sharing. @sws2apps/ui-ux-design: would you like to take a look at this for consideration please?

**Reply by ux-git** [reply]
Thanks @ratara5 for sharing your cleaning schedule example. Could you please share some more details of what each column means and maybe something else what may be useful for us while creating the cleaning scheduling functionality?

Thanks in advance!

**Comment by fschein** [comment]
+1. This function would be much appreciated, with an indicator of conflicts with other designs.
